### PR TITLE
Ontotools vm staging

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
How did you do this LFS thing with the larger ontology files?

**Comment by matentzn** [comment]
@falquaddoomi I think we can merge this at your own discretion. I added phenio and vbo, let me know if this works. :) Maybe just deploy this for now, and we worry about how to streamline the redeploy process in a subsequent step.

**Comment by falquaddoomi** [comment]
Hey @matentzn, I was able to import `vbo`, but I'm seeing thousands of the following errors for `phenio` in the log, and the import never seems to complete:

```
22:04:19.720 [main] WARN  u.a.e.s.o.l.ELKOWLOntologyLoader fillAllRelatedParents - Cycle detected where http://purl.obolibrary.org/obo/CARO_0001008 is related to one of its decendants
```

Any idea what might be going on? Thanks, and sorry for the delay.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,84 @@
+URIBASE = http://purl.obolibrary.org/obo
+# ONTS = upheno2 geno upheno_patterns hp chr mondo_patterns uberon-human-view
+ONTS = geno upheno_patterns hp chr mondo_patterns uberon-human-view
+#monarch
+ONTFILES = $(foreach n, $(ONTS), ontologies/$(n).owl)
+VERSION = "0.0.3" 
+IM=monarchinitiative/monarch-ols
+OLSCONFIG=/opt/ols/ols-config.yaml
+
+docker-build:
```
[the remark this answers is not in the document]
Comment (falquaddoomi) [reply Makefile:10]: Right, they're not needed. I took this Makefile from https://github.com/monarch-initiative/ols_monarch/blob/master/Makefile verbatim, with the attitude that I'd change as little as possible to get the ontologies importing. You make a good point that it should be cleaned up; I'll do so now.

File: config/ols-config/ols-config.yaml [hunk 2]
```diff
@@ -1,37 +1,151 @@
...
+    ontology_purl : file:/mnt/ontologies/vbo.owl
+  - id: phenio
+    preferredPrefix: PHENIO
+    description: >
+      An ontology for accessing and comparing knowledge concerning phenotypes across species and genetic backgrounds.
+    title: Phenomics Integrated Ontology
+    uri: https://github.com/monarch-initiative/phenio/releases/latest/download/phenio.owl
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+    reasoner: EL
```
Comment (matentzn) [inline config/ols-config/ols-config.yaml:149]: remove this row and see if this sorts out your cycle issue?

Comment (falquaddoomi) [reply config/ols-config/ols-config.yaml:149]: I attempted to remove lines 147-149 as you mentioned above, but I still got those cycle warnings when importing `phenio`. Were you suggesting just removing those three lines, just the line that says `reasoner: EL`, or were you suggesting removing the entire `phenio` block?

Comment (falquaddoomi) [reply config/ols-config/ols-config.yaml:149]: Just FYI, I'm currently trying to import phenio after removing all lines in `phenio.owl` that contain `CARO_0001008`, just as a test. It's been running for ~16 hours now, but at least there aren't any cycle warnings in the log. I'm going to let it run for around 24 hours, but do let me know if you have an idea of how long you expect it to take.

Comment (matentzn) [reply config/ols-config/ols-config.yaml:149]: This is probably not worth it. We need a much faster deployment, so I assume we need a uncycled phenio. You can comment out the hole block and I will go back to the drawing board! Thanks for checking!

**Pull request merged**